config/settings.yml

```
@@ -0,0 +1,4 @@
+max_length:
```

```suggestion
length:
```
ri thôi em, để sau còn tái sử dụng

ngoài ra, thêm end-line cho file này nhé

---

Dạ chị ạ